bundles/model/org.eclipse.smarthome.model.item/src/org/eclipse/smarthome/model/Items.xtext

```diff
@@ -39,7 +39,26 @@ ModelItemType :
 ;
 
 ModelBinding:
-	type=ID '=' configuration=STRING
+	type=ID '=' configuration=STRING   
+	('['
+        properties+=ModelProperty? (',' properties+=ModelProperty)*
+    ']')?
+;
+
+ModelProperty:
+    key=ID '=' value=ValueType
+;
+
+ValueType returns ecore::EJavaObject:
+    STRING | NUMBER | BOOLEAN
+;
+
+BOOLEAN returns ecore::EBoolean: 
+    'true' | 'false'
+;
+
+NUMBER returns ecore::EBigDecimal:
+    ID ('.' ID )?
```

Comment:
A number can contain any char from `ID`, so also `a-z`, `A-Z`, `_`?

Reply:
Stolen from [Thing.xtext](https://github.com/eclipse/smarthome/blob/09d8095efe9933216874e488b379cd9a10aa85e2/bundles/model/org.eclipse.smarthome.model.thing/src/org/eclipse/smarthome/model/thing/Thing.xtext#L88-L90). Why should it be different?

Reply:
@triller-telekom Think about numbers like `-4.52E+6` and you will understand why it is that way :-)

Reply:
@kaikreuzer Thank you for this explanation, but I still don't understand why you allow different characters than `e` or `E` and `_` for a number. Or is it just out of convenience reasons that you do not want to define a slightly different `ID` terminal? :)